.github/workflows/ci.yaml

```diff
@@ -256,7 +256,7 @@ jobs:
           git diff --exit-code charts/ingress-nginx/README.md
 
       - name: Run tests
-        run: helm unittest charts/ingress-nginx
+        run: helm unittest charts/ingress-nginx -f "tests/**/*_test.yaml"
```

Comment:
This way you're no longer running tests for resources directly in the `tests/` directory.

Reply:
That's not true actually, it's a glob expression, "**" matches any directory and its subdirectories, see the log when running this, it catches all test files now:

```
helm unittest charts/ingress-nginx -f "tests/**/*_test.yaml"

### Chart [ ingress-nginx ] charts/ingress-nginx

 PASS  Controller > ConfigMap > Add Headers	charts/ingress-nginx/tests/controller-configmap-addheaders_test.yaml
 PASS  Controller > ConfigMap > Proxy Headers	charts/ingress-nginx/tests/controller-configmap-proxyheaders_test.yaml
 PASS  Controller > ConfigMap	charts/ingress-nginx/tests/controller-configmap_test.yaml
 PASS  Controller > DaemonSet	charts/ingress-nginx/tests/controller-daemonset_test.yaml
 PASS  Controller > Deployment	charts/ingress-nginx/tests/controller-deployment_test.yaml
 PASS  Controller > HPA	charts/ingress-nginx/tests/controller-hpa_test.yaml
 PASS  Controller > IngressClass > Aliases	charts/ingress-nginx/tests/controller-ingressclass-aliases_test.yaml
 PASS  Controller > IngressClass	charts/ingress-nginx/tests/controller-ingressclass_test.yaml
 PASS  Controller > KEDA	charts/ingress-nginx/tests/controller-keda_test.yaml
 PASS  Controller > NetworkPolicy	charts/ingress-nginx/tests/controller-networkpolicy_test.yaml
 PASS  Controller > PodDisruptionBudget	charts/ingress-nginx/tests/controller-poddisruptionbudget_test.yaml
 PASS  Controller > PrometheusRule	charts/ingress-nginx/tests/controller-prometheusrule_test.yaml
 PASS  Controller > Service > Internal	charts/ingress-nginx/tests/controller-service-internal_test.yaml
 PASS  Controller > Service > Metrics	charts/ingress-nginx/tests/controller-service-metrics_test.yaml
 PASS  Controller > Service > Webhook	charts/ingress-nginx/tests/controller-service-webhook_test.yaml
 PASS  Controller > Service	charts/ingress-nginx/tests/controller-service_test.yaml
 PASS  Controller > ServiceMonitor	charts/ingress-nginx/tests/controller-servicemonitor_test.yaml
 PASS  Default Backend > Deployment	charts/ingress-nginx/tests/default-backend-deployment_test.yaml
 PASS  Default Backend > Extra ConfigMaps	charts/ingress-nginx/tests/default-backend-extra-configmaps_test.yaml
 PASS  Default Backend > PodDisruptionBudget	charts/ingress-nginx/tests/default-backend-poddisruptionbudget_test.yaml
 PASS  Default Backend > Service	charts/ingress-nginx/tests/default-backend-service_test.yaml
 PASS  Admission Webhooks > ValidatingWebhookConfiguration	charts/ingress-nginx/tests/admission-webhooks/validating-webhook_test.yaml
 PASS  Admission Webhooks > Patch Job > ClusterRole	charts/ingress-nginx/tests/admission-webhooks/job-patch/clusterrole_test.yaml
 PASS  Admission Webhooks > Patch Job > ClusterRoleBinding	charts/ingress-nginx/tests/admission-webhooks/job-patch/clusterrolebinding_test.yaml
 PASS  Admission Webhooks > Patch Job > Role	charts/ingress-nginx/tests/admission-webhooks/job-patch/role_test.yaml
 PASS  Admission Webhooks > Patch Job > RoleBinding	charts/ingress-nginx/tests/admission-webhooks/job-patch/rolebinding_test.yaml
 PASS  Admission Webhooks > Patch Job > ServiceAccount	charts/ingress-nginx/tests/admission-webhooks/job-patch/serviceaccount_test.yaml

Charts:      1 passed, 1 total
Test Suites: 27 passed, 27 total
Tests:       116 passed, 116 total
Snapshot:    0 passed, 0 total
Time:        919.615708ms
```

Reply:
Can also see it in this PR's CI log under "Run Tests" - https://github.com/kubernetes/ingress-nginx/actions/runs/11573205379/job/32214817319?pr=12257

Reply:
That's interesting because I tested the exact same locally and it did not work! 😃

Reply:
Okay, my fault. I just updated my Helm Unit Test plugin, now it works.

Reply:
Huh, maybe related to your shell interpreting the wildcard weird? 🤷🏻‍♂️